### DISCUSSION TITLE
Possibility to generate route redirects dynamically?

### DISCUSSION THREAD

**Discussion by procrates** [nuxt/nuxt#20486] [Questions]
Anyone knows if it is possible create routeRules dynamically from a database?

Spesifically need redirects when slugs changes. Optimal solution would be to have a collection of redirects and dynamically add the to nuxt.config routeRules.

Possible?

Thanks!

**Comment by maou-shonen** [comment]
Does global middleware and page:xxx hooks not meet your use case?
